### ISSUE TITLE
Unable to open draft from mobile

### ISSUE THREAD

**Issue by michalcharvat** [Intermesh/groupoffice#1106]
Since 6.7? there is no option open drafts on mobile.

Workaround - EmailClient.js:616

```
grid.on('navigate', function (sm, rowIndex, r) {
    if(GO.util.isMobileOrTablet() && (this.messagesGrid.store.reader.jsonData.drafts || this.messagesGrid.store.reader.jsonData.sent)) {
        GO.email.showComposer({
            uid: r.data['uid'],
            task: 'opendraft',
            template_id: 0,
            mailbox: this.mailbox,
            account_id: this.account_id
        });
    } else {
        if (r.data['uid'] != this.messagePanel.uid) {
            this.messagePanel.loadMessage(r.data.uid, r.data['mailbox'], this.account_id);
            this.messagePanel.show();
            if (!r.data.seen && this.messagesGrid.store.reader.jsonData.permission_level > GO.permissionLevels.read) {
                //set read with 2 sec delay.
                this.markAsRead.defer(2000, this, [r.data.uid, r.data['mailbox'], this.account_id]);
            }
        }
    this.messagePanel.show();
    }
},this)
```

**Comment by michalcharvat** [comment]
@mschering Shall I create PR for it? Do you plan any fix / workaround for 6.7 + 6.8? I assume new email module will be available in 6.11 which will be released ... tomorrow? :-D

**Comment by mschering** [comment]
There is an edit button now in 6.8.
